warrior/Actions/CommonActions/common_actions.py

```diff
@@ -165,11 +173,13 @@ def verify_data(self, expected, object_key, type='str', comparison='eq'):
                                                      type, comparison)
         if result == "FALSE":
             print_error("Expected: {0} {1} {2} but found {0}={3}".format(
-                object_key, comparison, expected, value))
+                        object_key, comparison, expected, value))
         if result == "TRUE":
             return True
-        else:
+        elif result == "FALSE":
```

Comment:
Hi @sugan19 ,

the ```result == 'FALSE'``` block is repeated on line 174 and 179. Please merge those blocks.

Reply:
@sanikakulkarni Addressed your comments